tasks.py

```diff
@@ -16,41 +16,75 @@
     Collection(
         common.tasks.setup_labels,
     ),
-    name='github',
+    name="github",
 )
 
 namespace.add_collection(
     Collection.from_module(
         common.dockerfiles.tasks,
         config={
-            'container_prefix': 'community',
+            "container_prefix": "community",
         },
     ),
-    name='docker',
+    name="docker",
 )
 
+
 @task
 def docs(ctx, regenerate_config=False, push=False):
     """Pull and push translations to Transifex for our docs"""
-    with ctx.cd(os.path.join(ROOT_PATH, 'docs')):
+    with ctx.cd(os.path.join(ROOT_PATH, "docs")):
         # Update our translations
-        ctx.run('tx pull -a')
+        ctx.run("tx pull -a")
         # Update resources
         if regenerate_config:
-            os.remove(os.path.join(ROOT_PATH, 'docs', '.tx', 'config'))
-            ctx.run('sphinx-intl create-txconfig')
-        ctx.run('sphinx-intl update-txconfig-resources --transifex-project-name readthedocs-docs')
+            os.remove(os.path.join(ROOT_PATH, "docs", ".tx", "config"))
+            ctx.run("sphinx-intl create-txconfig")
+        ctx.run(
+            "sphinx-intl update-txconfig-resources --transifex-project-name readthedocs-docs"
+        )
         # Rebuild
-        ctx.run('sphinx-intl build')
-        ctx.run('make gettext')
+        ctx.run("sphinx-intl build")
+        ctx.run("make gettext")
         # Push new ones
         if push:
-            ctx.run('tx push -s')
+            ctx.run("tx push -s")
 
 
 namespace.add_collection(
     Collection(
         docs,
     ),
-    name='l10n',
+    name="l10n",
+)
+
+
+@task(
+    help={
+        "package": "If specified, only update this package",
+    }
+)
+def update(ctx, package=None):
+    """Update Python dependencies from requirements/*.txt."""
+    cmd = ["pip-compile", "--resolver=backtracking"]
+    if package:
+        cmd.extend(["--upgrade-package", package])
+    else:
+        cmd.append("--upgrade")
+    files = [
+        "requirements/pip",
+        "requirements/docker",
+        "requirements/testing",
+        "requirements/docs",
+        "requirements/deploy",
+    ]
+    for file in files:
+        ctx.run(" ".join(cmd) + f" --output-file {file}.txt {file}.in")
+
+
+namespace.add_collection(
+    Collection(
+        update,
+    ),
+    name="deps",
```

Comment:
```suggestion
    name="requirements",
```

I would call it `requirements` (e.g. `inv requirements.update`) since it only update Python requirements.

Reply:
:+1: "requirements" does seem a lot easier to put into muscle memory :)